### ISSUE TITLE
Current query highlighting/selection settings have no effect

### ISSUE THREAD

**Issue by ygoe** [pawelsalawa/sqlitestudio#4761]
### Details
Pressing F9 with the cursor on one of multiple queries in the editor, only the query under the cursor will be executed. This is good! As visual feedback, that query is then selected in the editor. This is annoying because quick edits to a query always require moving the cursor to the position to be changed.

I found the option "Highlight current query" and unchecked it. But the editor selection behaviour goes on. Restarting the application doesn't change that.

Additionally, the syntax highlighting colour "Current query background" has no effect at all. It's not visible anywhere before or after executing a query.

I'm wondering if this has never been implemented, or tested, or the labels are all just very ambiguous and I got it all wrong. My expectations just come from obvious interpretations of the labels and actually observed situations within the first few minutes of using this for today.

### Steps to reproduce
1. Disable the option "Highlight current query"
2. Type multiple queries in an SQL editor
3. Execute one by pressing F9 (or other configured shortcut)

### Operating system
Windows 10

### SQLiteStudio version
3.4.4

**Comment by pawelsalawa** [comment]
The selection after execution is not the "highlighting" mentioned in configuration. This selection after execution is actually a hardcoded behavior, which now I agree that could be annoying. I will probably remove it.

Regarding the highlighting of current query, it's supposed to look like this:
![image](https://user-images.githubusercontent.com/481846/231256825-7cbc3c99-9fa3-41cd-9ebc-c534d5df0802.png)

The problem is that with any dark theme, the default "current query" color is barely any lighter than the backround, so it's hard to distinguish it:
![image](https://user-images.githubusercontent.com/481846/231256964-23e54c01-bca5-47c7-b81d-d08ed771a854.png)

It really is a bit lighter, but the difference is way to small. This has to be fixed.

Now, you can of course configure your own color, right? Well, you should, but apparently this does not work. Another thing to be fixed here.

So, in summary, two bugs to be fixed with this issue.

**Comment by pawelsalawa** [comment]
It should determine what color would it be, but unfortunately there is a bug, that causes the selected custom color to be ignored.
If it's left as default, the color will be "a bit darker than default editor background" if the overall theme is bright, or if the theme is dark, the default color of current query will be "a bit lighter than default editor backround", but as I've shown on pictures above, it's not very good for the dark theme, because the contrast is way too small.
